packages/react/src/components/organisms/TabContainer/tab-body.js

```diff
@@ -23,15 +27,19 @@ class TabBody extends React.Component {
         });
       });
     }
+    this.observer.observe(document, { attributes: true, childList: true, subtree: true });
   }
 
   componentDidMount() {
     const nodeList = document.getElementById(this.props.tabContainerBodyId);
     if (nodeList) {
+      console.log('rendered!!!!!')
```

Comment:
this was never triggered in SSR because nodeList is null on intial mount as the component has not been rendered yet.